api/src/plugins/growth-book.ts

```diff
@@ -2,9 +2,20 @@ import { GrowthBook, Options } from '@growthbook/growthbook';
 import { FastifyPluginAsync } from 'fastify';
 import fp from 'fastify-plugin';
 
+declare module 'fastify' {
+  interface FastifyInstance {
+    gb: GrowthBook;
+  }
+}
+
 const growthBook: FastifyPluginAsync<Options> = async (fastify, options) => {
   const gb = new GrowthBook(options);
-  await gb.init({ timeout: 3000 });
+  const res = await gb.init({ timeout: 3000 });
+
+  if (res.error) {
+    fastify.log.error('Failed to initialize GrowthBook');
```

Comment:
Just so we do not **have** to go to Sentry to find the specific reason behind the error.

```suggestion
    fastify.log.error({ error: res.error }, 'Failed to initialize GrowthBook');
```